walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,177 @@
 
 package com.google.sps;
 
-import java.util.Collection;
+import java.util.*;
 
 public final class FindMeetingQuery {
+
+    int firstEventStartTime;
+    int firstEventEndTime;
+    int secondEventStartTime;
+    int secondEventEndTime;
+
+  // The query function returns free time ranges that attendees might have imbetween meetings or 
+  // after meetings.
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    throw new UnsupportedOperationException("TODO: Implement this method.");
+    Collection<TimeRange> avaliableTime;
+
+    List<TimeRange> listTransfer = new ArrayList<TimeRange>();
```

Comment:
Can you give this variable a better name? Not quite sure what it's supposed to be for.